key.ui/src/main/java/de/uka/ilkd/key/logging/JsonLayout.java

```diff
@@ -87,7 +86,11 @@ public String doLayout(ILoggingEvent event) {
         printEntry(pw, "thread", event.getThreadName());
         printEntry(pw, "level", event.getLevel().levelStr);
         printEntry(pw, "loggerName", event.getLoggerName());
-        printEntry(pw, "message", event.getFormattedMessage().replace("\\","\\\\"));
+        printEntry(pw, "message", event.getFormattedMessage()
+                .replace("\\", "\\\\")
+                .replace("\"", "\\\"")
+                .replace("\n", "\\n")
```

Comment:
This looks very suspicious. Either JSON supports line breaks in strings anyways (as far as i know this is the case) or this breaks as soon as any other line break than `\n` appears inside a log message.

Reply:
Did you check this with any other line breaks?